chapters/memory-layout/c-assembly-interaction/drills/tasks/export-fix/solution/a-func/Makefile

```diff
@@ -0,0 +1,3 @@
+PROGNAME := main
+include ../../../../utils/Makefile.generic
+
```

Comment:
`Makefile.generic` doesn't exist anymore. Switch this to a symlink to `../../support/a-func/Makefile`. Do this for all makefiles that contain `include`s and create standalone makefiles where they're missing.